pnpm-lock.yaml

```diff
@@ -1,9 +1,5 @@
 lockfileVersion: '6.0'
 
-settings:
-  autoInstallPeers: false
-  excludeLinksFromLockfile: false
-
```

Comment:
These settings shouldn't be removed.

Reply:
Oh, I see they are at the end, that is strange

Reply:
Maybe it is the version, using `corepack pnpm i` moves these lines back to the top for me.

Reply:
This worked much better with changes in the file.